### ISSUE TITLE
Update play area grid

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#222]
The play area grid is out of date from the latest designs.  It should be updated to match the latest designs in 

https://docs.google.com/spreadsheets/d/1BiXFN2dRWfsjqV2WvKAXnZFhsk0jCxbnT0gkZr_T5T0/edit?ts=568067c0#gid=102492439

From #215

**Comment by jessegreenberg** [comment]
I am going to work on this next as fixing this will probably resolve several open issues in this repo relateted to accessibility.

**Comment by terracoda** [comment]
@jessegreenberg, I have changed the actual grid position text, but not the model. See pull request #254. I hope this is helpful.

**Comment by jessegreenberg** [comment]
Here is the mockup, scaled to the simulation bounds:
![mockup](https://cloud.githubusercontent.com/assets/6396244/26370603/95e64182-3fb5-11e7-9bd6-28e4273862db.PNG)

Here is the new grid, in the sim:
![grid](https://cloud.githubusercontent.com/assets/6396244/26370647/ad0fbf3c-3fb5-11e7-9b7d-bbebf10ff0bb.PNG)

The grid matches pretty well, but other sim objects don't quite line up.  Here are the two on top of each other:
![overlay](https://cloud.githubusercontent.com/assets/6396244/26370710/e164ee42-3fb5-11e7-9967-309644b265ff.PNG)

Perhaps the mockup screenshot wasn't taken with the default layout dimensions.  Most noticeable is the balloon isn't centered in the "Center of PA" column and there is some extra space to the left of the PA.

**Comment by jessegreenberg** [comment]
If we shift the "Right Side of PA" column a bit to the right it looks more centered, thought areas dont' quite align with the mockup.  Is this OK @terracoda?

**Comment by jessegreenberg** [comment]
![new](https://cloud.githubusercontent.com/assets/6396244/26371126/fdd8e4ce-3fb6-11e7-84e3-3b1d229261e8.PNG)

**Comment by terracoda** [comment]
My mock-up was made from screen shots, and then adjusted in photoshop, more than once. It is not surprising that it might be off a little. 

@jessegreenberg, I think your adjustment looks good.

**Comment by jessegreenberg** [comment]
Great, thanks @terracoda.  The grid has been updated, but descriptions and hotkeys will not work correctly yet, I re-implemented the play area model.

**Comment by terracoda** [comment]
@jessegreenberg, yep understood.

**Comment by jessegreenberg** [comment]
Described grid has been fully implemented to match design docs. Closing.
